airflow/providers/microsoft/azure/log/wasb_task_handler.py

```diff
@@ -17,11 +17,13 @@
 # under the License.
 import os
 import shutil
+from typing import Optional, Tuple, Dict
 
 from azure.common import AzureHttpError
 from cached_property import cached_property
 
 from airflow.configuration import conf
+from airflow.models import TaskInstance
```

Comment:
I am afraid that this will not create cyclical imports. We've had a issue with this module before.

Reply:
I am sorry I didn't understand this!

Reply:
This will create cyclic import i.e. this module will call itself in a loop, causing errors

Reply:
mainly because this is a log handler. I think that is what @mik-laj means

Reply:
I would prefer not to add this type here if we do not have system tests that will detect regression.
See: https://github.com/apache/airflow/pull/4601
For other task handlers, we have system tests that can detect such a problem, but this handler is not covered by these types of tests.
See: 
https://github.com/apache/airflow/blob/master/tests/providers/google/cloud/log/test_gcs_task_handler_system.py
https://github.com/apache/airflow/blob/master/tests/providers/google/cloud/log/test_stackdriver_task_handler_system.py

Reply:
Oh okay, this will create a cyclic import! I was reading it as not create cyclic import so got confused. 

Yup, that totally makes sense, for now, I will remove this and later I will create system test for this. Thanks for sharing the example.